core/internal/testutils/testutils.go

```diff
@@ -116,15 +116,15 @@ func AfterWaitTimeout(t *testing.T) <-chan time.Time {
 }
 
 // Context returns a context with the test's deadline, if available.
-func Context(t *testing.T) context.Context {
+func Context(t testing.TB) context.Context {
```

Comment:
Maybe s/t/tb this var so just `t` can be used below? :shrug: 
```suggestion
func Context(tb testing.TB) context.Context {
```

Reply:
Fixed